build.yaml

```diff
@@ -0,0 +1,15 @@
+targets:
+  $default:
+    builders:
+      build_vm_compilers|entrypoint:
+        generate_for:
+          include:
+            - "test/unit/**.dart.vm_test.dart"
+      build_web_compilers|entrypoint:
+        generate_for:
+          include:
+            - "test/unit/**.dart.browser_test.dart"
+            - "example/**.dart"
```

Comment:
Small packages don't really benefit from this configuration (it should work with the defaults, and this isn't going to improve build perf), so my preference would be to omit it. We're also not currently running tests in dart2js anywhere in this repo, so we could remove the `dart2js_args` portion below, as well, or we could leave it for convenience if someone did need to run them in release mode.

Reply:
Unfortunately we can't remove the whole thing, I can cut it down to 
```
targets:
  $default:
    builders:
      build_vm_compilers|entrypoint:
        generate_for:
          include:
            - "test/unit/**.dart.vm_test.dart"
``` 
otherwise we'll get
```
[WARNING]build_vm_compilers:entrypoint on example/example.dart: Skipping compiling opentracing|example/example.dart for the vm because some of its
transitive libraries have sdk dependencies that not supported on this platform:

opentracing|example/example.dart

https://github.com/dart-lang/build/blob/master/docs/faq.md#how-can-i-resolve-skipped-compiling-warnings
```